src/components/MenuItem/MenuItem.jsx

```diff
@@ -46,19 +58,33 @@ export class MenuItem extends React.Component {
 
   action() {
     const { addToOrder, name } = this.props;
-    const { total, bread, isHot } = this.state;
+    const { total, bread, isHot, owner } = this.state;
 
     addToOrder(
       name,
       total,
       bread,
-      isHot
+      isHot,
+      owner
     );
   }
 
+  handleClick() {
```

Comment:
Might be worth renaming this to match the closing modal function 👍 

`openModal` and `closeModal`, or you could have a `toggleModal` which just inverts its current state